tests/zero_code_change/test_tensorflow2_integration.py

```diff
@@ -50,7 +51,9 @@ def helper_test_keras_v2(script_mode: bool = False, eager_mode: bool = True):
     """ Test the default ZCC behavior of saving losses and metrics in eager and non-eager modes."""
     smd.del_hook()
     tf.keras.backend.clear_session()
-    if not eager_mode:
+    if not eager_mode and is_tf_2_3() is False:
```

Comment:
is this required in the other unit tests as well or something unique to ZCC?

Reply:
This isn't required in other unit tests because they run in script mode.

zero_code_change tests need changes to AWS TF, which currently supports basic ZCC functionality